package.json

```diff
@@ -20,7 +20,8 @@
     "@testing-library/dom": "10.1.0",
     "@types/react": "18.2.0",
     "@types/react-dom": "18.2.0",
-    "jackspeak": "2.1.1"
+    "jackspeak": "2.1.1",
+    "@leafygreen-ui/hooks": "8.1.3"
```

Comment:
Did you try only forcing this version for combobox? I believe this would be the syntax:
```json
"@leafygreen-ui/combobox/@leafygreen-ui/hooks"
```

It would be preferable to not apply this to all LG components I think if it works

Reply:
Oh yes, I could only get this working by moving the content to Spruce `package.json`, but I assume that's fine